Documentation/ceph-upgrade.md

```diff
@@ -89,6 +89,40 @@ you should change to match where your images are located.
         value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
 
+### Retroactively enabling volume expansion
+
+When upgrading to the CSI 2.0 driver with existing persistent volumes, the following considerations should be taken into account when choosing to leverage the volume expansion feature:
+
+#### Remediate existing storage classes
+
+It is also necessary to ensure that the controlling StorageClass have the required changes
+to support Volume Expansion see the documentation about configuring [dynamically expanding volumes](https://rook.io/docs/rook/master/ceph-csi-drivers.html#dynamically-expand-volume).
```

Comment:
For links to rook docs, specify the following format and the correct full link will be generated when the docs are published. (note the `.md` extension that is translated as well)
```
[dynamically expanding volumes](ceph-csi-drivers.md#dynamically-expand-volume)
```

Reply:
This should now be resolved!